src/main/cliapp/src/containers/experimentalConditionsPage/ExperimentalConditionsTable.js

```diff
@@ -240,23 +240,25 @@ export const ExperimentalConditionsTable = () => {
       }
     };
 
-  const conditionGeneOntologyEditorTemplate = (props) => {
-      return (
-          <>
-            <ConditionGeneOntologyEditor
-                autocompleteFields={["curie", "name", "crossReferences.curie", "secondaryIdentifiers", "synonyms"]}
-                rowProps={props}
-                searchService={searchService}
-                setExperimentalConditions={setExperimentalConditions}
-            />
-              <ErrorMessageComponent
-                errorMessages={errorMessages[props.rowIndex]}
-                errorField={"conditionGeneOntology"}
-            />
-          </>
-      );
+  const singleOntologyEditorTemplate = (props, fieldname, endpoint) => {
+    return (
+      <>
+        <SingleOntologyEditor
+          autocompleteFields={["curie", "name", "crossReferences.curie", "secondaryIdentifiers", "synonyms"]}
```

Comment:
The only issue I see with this is that these fields might want to be passed in rather than hardcoded here.

Reply:
@oblodgett - done in latest commit